src/commands/new-proj.ts

```diff
@@ -20,20 +28,86 @@ async function action() {
     if (nameResult.name) {
       projName= nameResult.name;
     } else {
-      throw new Error("ERROR: No name was given for the project");
+      throw new Error("No name was given for the project");
+    }
+
+    const typeResult = await this.prompt({
+      type : 'list',
+      name : 'type',
+      choices : ["Quarkus", "React", "No framework"],
+      message : "Framework for the project: "
+    });
+
+    if (typeResult.name) {
+      projType= typeResult.name;
+    } else {
+      throw new Error("No type was given for the project");
+    }
+
+    const pathResult = await this.prompt({
+      type : 'input',
+      name : 'path',
+      message : "Set a path where to initiate repository, leave empty for default: "
+    });
+
+    if (pathResult?.path) {
+      givenPath = pathResult.path;
     }
   } catch (err) {
-    throw new Error(err);
+    throw new Error("Error: " + err);
+  }
+
+  givenPath = await PathUtils.fixPath(givenPath);
```

Comment:
awaits should preferrably have try-catch

Reply:
they now have try-catch